longestConsecutiveSequence.py

```diff
@@ -0,0 +1,27 @@
+class Solution(object):
+    def longestConsecutive(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        summary = {}
+        res = 0
+        for num in nums:
+            if num not in summary:
+                summary[num] = 1
+        for num in summary:
+            if summary[num] == 0:
+                continue
+            ans = 1
+            j = 1
+            while num + j in summary:
```

Comment:
Maybe think of a way to avoid code duplication, 17 - 20 and 22 - 25 are almost the same.